pip_audit/_cli.py

```diff
@@ -287,6 +287,14 @@ def _parser() -> argparse.ArgumentParser:
         # argparse's default renderer uses __repr__ and produces
         # a pretty unpleasant help message.
     )
+    parser.add_argument(
+        "--ignore-vuln",
+        type=str,
+        action="append",
+        dest="ignore_vulns",
+        default=[],
+        help="ignore a specific vulnerability by its PYSEC id",
```

Comment:
This is no longer just PYSEC IDs, correct? (Also, please capitalize 'ID' here)

Reply:
Yeah, this should just be `by its ID`.